test/image/mocks/bar_text_horizontal_group.json

```diff
@@ -0,0 +1,23 @@
+{
+    "data":[
```

Comment:
@n-riesco general comment: would be adding the `text` attribute and friends to existing mocks you added for `width` / `offset` / `base`? We have over 300 image mocks now - which take about 2 and 1/2 minute to tests. It's time to be more meticulous about adding new mocks.